Unable to compile on any platform (issue shared with other NodeJS bindings)

```  CC       sodium/libsodium_la-utils.lo
  CC       sodium/libsodium_la-version.lo
  CC       randombytes/salsa20/libsodium_la-randombytes_salsa20_random.lo
  CC       randombytes/sysrandom/libsodium_la-randombytes_sysrandom.lo
  CC       crypto_scalarmult/curve25519/donna_c64/libsodium_la-curve25519_donna_c64.lo
make[3]: *** No rule to make target 'crypto_stream/salsa20/xmm6/salsa20_xmm6-asm.S', needed by 'crypto_stream/salsa20/xmm6/libsodium_la-salsa20_xmm6-asm.lo'.  Stop.
make[3]: Leaving directory '/app/node_modules/sodium-native/libsodium/src/libsodium'
make[2]: Leaving directory '/app/node_modules/sodium-native/libsodium/src/libsodium'
make[2]: *** [Makefile:2925: install-recursive] Error 1
make[1]: *** [Makefile:397: install-recursive] Error 1
make[1]: Leaving directory '/app/node_modules/sodium-native/libsodium/src'
make: *** [Makefile:511: install-recursive] Error 1
/app/node_modules/sodium-native/preinstall.js:93
        if (err) throw err
                 ^

Error: make exited with 2
    at ChildProcess.<anonymous> (/app/node_modules/sodium-native/preinstall.js:129:25)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

I can compile libsodium-stable though, so I'm not sure what's up with that.

--- a/package.json
+++ b/package.json
@@ -23,7 +23,9 @@
     "test": "standard && tape \"test/*.js\"",
     "install": "node-gyp-build \"node preinstall.js\" \"node postinstall.js\"",
     "prebuild": "prebuildify -a --strip --preinstall \"node preinstall.js\" --postinstall \"node postinstall.js\"",
-    "prebuild-ia32": "prebuildify -a --strip --preinstall \"node preinstall.js\" --postinstall \"node postinstall.js\" --arch=ia32"
+    "prebuild-ia32": "prebuildify -a --strip --preinstall \"node preinstall.js\" --postinstall \"node postinstall.js\" --arch=ia32",
+    "prepack": "cd libsodium && git reset --hard && git clean -fd && echo '' > .gitignore",
+    "postpack": "cd libsodium && git reset --hard"
   },
   "repository": {
     "type": "git",

